packagedb/management/commands/watch_packages.py

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# PurlDB is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/purldb for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+from commoncode import cliutils
+from django.core.management.base import BaseCommand
+from fetchcode.package_versions import SUPPORTED_ECOSYSTEMS
+from fetchcode.package_versions import versions
+from minecode.models import PriorityResourceURI
+from packagedb.models import Package
+from packageurl import PackageURL
+from univers.version_range import RANGE_CLASS_BY_SCHEMES
+
+VERSION_CLASS_BY_PACKAGE_TYPE = {
+    pkg_type: range_class.version_class
+    for pkg_type, range_class in RANGE_CLASS_BY_SCHEMES.items()
+}
+
+PRIORITY_QUEUE_SUPPORTED_ECOSYSTEMS = ["maven", "npm"]
+
+
+class Command(BaseCommand):
+    help = "Watch the packages for their latest versions and add them to the priority queue for scanning and indexing."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--purl", type=str, help="Specify a PURL to watch single package."
+        )
+
+    def handle(self, *args, **options):
```

Comment:
@keshav-space Now that #271 has been merged, can you take a look at https://github.com/nexB/purldb/blob/50ac7dfbf1f7e4452a8f8711c1abe15ad7c16607/packagedb/tasks.py#L26 to see if there is a way to be able to use/modify this function here, just to avoid code duplication

Reply:
Done :)